Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/DeltaWizard/PomDeltaSettingsWizardPage.xaml.cs

```diff
@@ -138,12 +153,22 @@ private void SetElementLocatorsSettingsGridView()
             xElementLocatorsSettingsGrid.SetTitleStyle((Style)TryFindResource("@ucTitleStyle_4"));
         }
 
-        private void CheckUnCheckAllElements(object sender, RoutedEventArgs e)
+        private void CheckUnCheckAllBasicElements(object sender, RoutedEventArgs e)
+        {
+            if (mWizard.mPomDeltaUtils.PomLearnUtils.AutoMapBasicElementTypesList.Count > 0)
+            {
+                bool valueToSet = !mWizard.mPomDeltaUtils.PomLearnUtils.AutoMapBasicElementTypesList[0].Selected;
+                foreach (UIElementFilter elem in mWizard.mPomDeltaUtils.PomLearnUtils.AutoMapBasicElementTypesList)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5763742)